LandSandBoat/server
# [QUEST]Sandoria Teleport Scroll Quest to IF

Converts Healing the Land and Sorcery of the North to Interaction Framework.



**_I affirm:_**

## What does this pull request do?
-Converts the following questline to Interaction Framework.
    Healing the Land - Teleport Holla
    Sorcery of the North - Teleport Vhazl
- Removes NPC's Dialog for associated NPC's and moved logic into Interaction Framework files.
- Adds defaultActions for those NPC that did not have it that were associated with the questline.
- Adds mustZone between Healing the Land and Sorcery of the North.
[Healing the Land Capture](https://www.youtube.com/watch?v=rMuwWGadL04)
[Sorcery of the North Capture](https://www.youtube.com/watch?v=lq4Z9WRTjds)


## Steps to test these changes
Healing the Land:
Speak with Eperdur in Northern Sandoria with 4 Fame or higher.
Quest is auto accepted (no option to decline)
Travel to Gusgen Mines to QM3 
Place the KI 
Return to Eperdur for Teleport Holla scroll.
Must zone between next quest.
Speak to Eperdur after zone.
Quest is auto accepted (no option to decline)
Acquire check key from Feiyin - open chest key to get KI FIEYIN_MAGIC_TOME
Return to Eperdur with KI to receive reward of Teleport Vhazl scroll.
